cross/build-rootfs.sh

```diff
@@ -84,6 +84,20 @@ for i in "$@" ; do
                 __UbuntuCodeName=xenial
             fi
             ;;
+        jessie)
+            __UbuntuCodeName=jessie
+            __UbuntuRepo="http://ftp.debian.org/debian/"
+            ;;
+        tizen)
+            if [ "$__BuildArch" != "arm-softfp" ]; then
```

Comment:
@gkhanna79 Thanks!  I've updated it. And also change error message to "Unsupported target platform." as you suggested in CoreCLR.